pkg/log/zap/zap_test.go

```diff
@@ -296,3 +299,170 @@ var _ = Describe("Zap logger setup", func() {
 		})
 	})
 })
+
+var _ = Describe("Zap log level flag options setup", func() {
+	var (
+		fromFlags Options
+		fs        flag.FlagSet
+		info      = "info text"
+		debug1    = "debug 1 text"
+		debug2    = "debug 2 text"
+		debug3    = "debug 3 text"
```

Comment:
```suggestion
		logLevel1<Type>    = "output to test log level 1 (type)"
		logLevel2   = "output to test log level 2 (type)"
		logLevel3   = "output to test log level 3 (type)"
```
WDYT? would not make more clear what it has been testing? 
Could we add the type (info/debug and etc) in each level to make clear?

Reply:
I am not sue I understood the concern in this statement. 
"Also, we have in levelStrings just 3 levels mapped, however, here we have info + 3 others levels. is not info a level as well? What does mean info?"

Reply:
What means debug1 or 2 or 3? What is the level(error/info) related to each one? It is not clear, and because of this, I'd suggest you add the type(error, info) in the variable name in order to be easier its understanding. 

Also, we have only info and the error level why we have info, debug1, debug2, debug3?